apps/extension/src/ui/apps/dashboard/routes/Portfolio/PortfolioAssets.tsx

```diff
@@ -102,44 +102,44 @@ const PageContent = ({ balances }: { balances: Balances }) => {
               {account && (
                 <ContextMenu placement="bottom-end">
                   <ContextMenuTrigger className="hover:bg-grey-800 text-body-secondary hover:text-body flex h-[1.5em] w-[1.5em] flex-col items-center justify-center rounded-full text-lg">
-                    <IconMore />
+                    <MoreHorizontalIcon />
                   </ContextMenuTrigger>
                   <ContextMenuContent className="border-grey-800 z-50 flex w-min flex-col whitespace-nowrap rounded-sm border bg-black px-2 py-3 text-left text-sm shadow-lg">
+                    {canToggleIsPortfolio && (
+                      <ContextMenuItem onClick={toggleIsPortfolio}>{toggleLabel}</ContextMenuItem>
+                    )}
                     {canSendFunds && (
                       <ContextMenuItem onClick={sendFunds}>{t("Send funds")}</ContextMenuItem>
                     )}
                     <ContextMenuItem onClick={copyAddress}>{t("Copy address")}</ContextMenuItem>
                     {showTxHistory && (
                       <ContextMenuItem onClick={browseTxHistory}>
-                        {t("Transaction History")}
+                        {t("Transaction history")}
                       </ContextMenuItem>
                     )}
                     {canRename && (
-                      <ContextMenuItem onClick={openAccountRenameModal}>
+                      <ContextMenuItem onClick={() => openAccountRenameModal()}>
```

Comment:
is there a reason for this change ?

Reply:
[Discussion here](https://github.com/TalismanSociety/talisman/pull/920#discussion_r1261991734)